types.go

```diff
@@ -123,7 +126,20 @@ func toUint(in interface{}) (uint64, error) {
 
 	switch inValue.Kind() {
 	case reflect.String:
-		return strconv.ParseUint(inValue.String(), 0, 64)
+		s := strings.TrimSpace(inValue.String())
+		if s == "" {
+			return 0, nil
+		}
+
+		// support the float input
+		if strings.Contains(s, ".") {
+			f, err := strconv.ParseFloat(s, 64)
+			if err != nil {
+				return 0, err
+			}
+			return uint64(f), nil
+		}
+		return strconv.ParseUint(s, 0, 64)
```

Comment:
these two cases let an empty column for a float or in be parsed as 0. Is this behavior we want? e.g.

```
col1,col2,col3
3,,5
10,11,12
```
if all fields are ints to be parsed as `3,0,5` and `10,11,12`, which looks like an error to me.